code/html/custom.js

```diff
@@ -1366,7 +1367,9 @@ function processData(data) {
         // -----------------------------------------------------------------------------
 
         if ("haConfig" === key) {
-            $("#haConfig").show();
+            websock.send("{}");
```

Comment:
Why this?

Reply:
Speeding up Windows <-> ESP tcp exchange. Quite noticeable.
https://github.com/xoseperez/espurna/pull/1387#issuecomment-446388717
https://github.com/xoseperez/espurna/issues/1300#issuecomment-432781707

Reply:
NOTE: in case DEBUG_SUPPORT is enabled, client must send something back too in response to the "weblog". Otherwise, haConfig messages are stuck behind. Gosh darnit...